src/tests/editor.test.js

```diff
@@ -768,7 +768,7 @@ describe('Editor', () => {
                 await testEditor(BasicEditor, {
                     contentBefore: '<h1>[<b>abcd</b></h1><p>ef]gh</p>',
                     stepFunction: deleteForward,
-                    contentAfter: '<h1>[]gh</h1>',
+                    contentAfter: '<h1><b>[]</b>gh</h1>',
```

Comment:
TrAdEoFf I guess ?

Reply:
Look again. This test has 4 expressions of the same situation and was expecting a different result in two of them. With my implementation we get the same result every time, which is the browser's default. This is an improvement, not a tradeoff.